lib/config.go

```diff
@@ -475,6 +475,37 @@ func (c *RuntimeConfig) Validate(onExecution bool) error {
 		if _, err := sysregistriesv2.GetRegistries(nil); err != nil {
 			return fmt.Errorf("invalid /etc/containers/registries.conf: %q", err)
 		}
+
+		for _, hooksDir := range c.HooksDir {
+			if _, err := os.Stat(hooksDir); err != nil {
+				return fmt.Errorf("invalid hooks_dir entry '%s'", hooksDir)
```

Comment:
Can we wrap `err`? Otherwise, we don't know which kind of error we're having (e.g., no file/dir or a permission one etc.).

Reply:
Sure, I adapt it as suggested.